plugins/woocommerce/client/admin/client/settings-payments/index.tsx

```diff
@@ -349,27 +326,27 @@ export const SettingsPaymentsOfflineWrapper = () => {
 				) }
 				context={ 'wc_settings_payments__offline_pms' }
 			/>
-			<Suspense
-				fallback={
-					<>
-						<div className="settings-payments-offline__container">
-							<div className="settings-payment-gateways">
-								<div className="settings-payment-gateways__header">
-									<div className="settings-payment-gateways__header-title">
-										{ __(
-											'Payment methods',
-											'woocommerce'
-										) }
-									</div>
-								</div>
-								<ListPlaceholder rows={ 3 } />
-							</div>
-						</div>
-					</>
-				}
-			>
-				<SettingsPaymentsOfflineChunk />
-			</Suspense>
+			<div className="settings-payments-offline__container">
+				<div className="settings-payments-offline__header">
+					<BackButton
+						href={ getNewPath( {}, '' ) }
+						title={ __(
+							'Return to payments settings',
+							'woocommerce'
+						) }
+						isRoute={ true }
+						from={ 'woopayments_payment_methods' }
+					/>
+					<h1 className="components-truncate components-text woocommerce-layout__header-heading woocommerce-layout__header-left-align">
+						<span className="woocommerce-settings-payments-header__title">
+							{ __( 'Take offline payments', 'woocommerce' ) }
+						</span>
+					</h1>
+				</div>
+				<Suspense fallback={ <ListPlaceholder rows={ 3 } /> }>
```

Comment:
The suspense is moved down here. This way, the non-moving UI is rendered first, while the UI that depends on API requests is encapsulated in the suspense.